### PR TITLE
Simplified Chinese Translation for TIC3 Early Publicity | 匠魂3简体中文翻译早期公示

### PR DESCRIPTION
**Pull request by ETStareak**
This Pull Request is for translation try out and check. It'll keep on updating, for coming alpha version of TIC3. Draft until TIC3 is nearly completed.

本PR公示匠魂3的游戏内容汉化，内容将会在匠魂3正式发布时实装。
考虑到匠魂3的革新性，本代汉化以直接的翻译为主，在一些著名内容上参考由craft everywhere制作的匠魂1和由酒石酸菌制作的匠魂2基本汉化，并进行贴合原版翻译方式的修正和优化，以保证新老玩家均能获得较好的阅读体验。

2021/2/10：完成了与匠魂2汉化的对接，绝大部分可由旧版本继承的内容均已汉化，并进行了相当幅度的优化整理。优化情况请参考#4173，该PR是为匠魂3进行的第一次汉化准备。
当前进度：汉化匠魂3的官方计划文档，并以此汉化匠魂3新增的内容。

### CODE REVIEW

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,1089 @@
...
+    "block.tconstruct.blue_magma_slime_grass": "岩浆黏性草方块",
+    "block.tconstruct.blue_magma_slime_grass.tooltip": "block.tconstruct.slime_grass.tooltip",
+    "block.tconstruct.purple_magma_slime_grass": "岩浆黏性草方块",
+    "block.tconstruct.purple_magma_slime_grass.tooltip": "block.tconstruct.slime_grass.tooltip",
+    "block.tconstruct.orange_magma_slime_grass": "岩浆黏性草方块",
+    "block.tconstruct.orange_magma_slime_grass.tooltip": "block.tconstruct.slime_grass.tooltip",
+
+    "block.tconstruct.slime_grass.tooltip": "好软啊！能让你跳得更高",
+
+    "block.tconstruct.blue_slime_leaves": "史莱姆树叶",
```
Comment (3TUSK) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json:907]: Either only use `黏液/黏性` or only use `史莱姆`, not mixing.

File: src/main/resources/assets/tconstruct/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,1089 @@
...
+    "block.tconstruct.blue_clear_stained_glass_pane": "蓝色染色通透玻璃板",
+    "block.tconstruct.brown_clear_stained_glass_pane": "棕色染色通透玻璃板",
+    "block.tconstruct.green_clear_stained_glass_pane": "绿色染色通透玻璃板",
+    "block.tconstruct.red_clear_stained_glass_pane": "红色染色通透玻璃板",
+    "block.tconstruct.black_clear_stained_glass_pane": "黑色染色通透玻璃板",
+
+    "block.tconstruct.soul_glass": "Soul Glass",
+    "block.tconstruct.soul_glass_pane": "Soul Glass Pane",
+
+    "block.tconstruct.glow": "荧光球",
```
Comment (3TUSK) [inline src/main/resources/assets/tconstruct/lang/zh_cn.json:95]: This is the *block*, not the item. IIRC it is the technical block created by tools w/ the glow modifier, i.e. when you light up an area using tools w/ that modifier, an invisible block is placed, which is this very block.

**Review by 3TUSK — COMMENTED** [review]
Manually curated (incomplete) diff list:
|             English (Old)             |         English (New)               |       S. Chinese (Old)    |        S. Chinese (New)    |
| ------------------------------------- | ----------------------------------- | ------------------------- | -------------------------- |
|                Grout                  |               Grout                 |            水泥           |            砖泥            |
|              I am Grout!              |            I am Grout!              |         我就是渣渣！      |        哪里需要哪里搬！    |
|          Slimy mud                    |             Slimy mud               |          黏滑泥泞         |           黏液泥           |
|      ${type} Slime Block              |            Slime Block              |       ${type}史莱姆块     |           黏液块           |
|  Congealed ${type} Slime Block        |       Congealed Slime Block         |     ${type}凝固史莱姆块   |         凝固黏液块         |
|  Clear ${color} Stained Glass (Pane)  | Clear ${color} Stained Glass (Pane) | 通透${color}染色玻璃（板）| ${color}染色通透玻璃（板） |
|          Glow                         |                Glow                 |           辉光            |           荧光球           |
|    Silky Block of Jewel               |            Silky Block of Jewel     |        精致珠宝块         |         裹绸宝石块         |
|       Slime Ball                      |            Slime Ball               |      ${type}史莱姆球      |           黏液球           |
|        Coagulated Blood               |                Coagulated Blood     |           凝血            |           凝血球           |
|    Gelatinous Slime Drop              |         Gelatinous Slime Drop       |  黏胶${type}史莱姆液滴    |           黏胶滴           |
|         Dried Brick                   |              Dried Brick            |    干燥粘土砖             |           干制砖           |
|      ${type} Slime Crystal            |           ${type} Slime Crystal     |    ${type}史莱姆水晶      |       ${type}黏液晶体      |
|      Don't skip leg day!<sup>1</sup>  |            Don't skip leg day!      |   *水平方向各拓展一格*（未直译）   |    别再在练腿日偷懒啦！    |
|       Do you even lift?<sup>1</sup>   |           Do you even lift?         |   *垂直方向各拓展一格*（未直译）    |你别是连身高都一起膨胀了吧？|
|         Reinforcement                 |           Reinforcement             |    耐久强化大板           |           加固板           |
|           Silky Cloth                 |              Silky Cloth            |           绢布            |            丝绢            |
|              Silky Jewel              |              Silky Jewel            |          精致珠宝         |          裹绸宝石          |
|             Necrotic Bone             |             Necrotic Bone           |          凋零之骨         |       噬生之骨             |
|          Creative Modifier            |          Creative Modifier          |      创造模式强化头颅     |       创造强化品           |
|Adds an additional modifier<sup>1</sup>|        Adds an additional modifier  |       增加一个强化槽      |    附加一个额外的强化槽    |
|            Seared Stone (the fluid)               |            Seared Stone             |         熔融焦黑石头      |          焦黑熔石          |
|             Stone Torch               |             Stone Torch             |           石柄火把        |          石棍火把          |
|            Stone Ladder               |            Stone Ladder             |             石梯          |          石棍梯子          |
|Wooden Rail|Wooden Rail|木质轨道|木轨|
|Wooden Rail with a Trapdoor|Wooden Dropper Rail|带活板门的木质轨道|活板门木轨|
|Minecarts with a Hopper can\ninsert into the block below|Minecarts with a Hopper can\ninsert into the block below|漏斗矿车可以\n穿过这个方块下面|漏斗矿车能通过它\n向下方的方块输出物品|
|Pointy! Also sticks to walls.          |      Pointy! Also sticks to walls.  |  非常尖锐！可以拿来做围墙  |   很尖锐！还能安装在墙上。   |
|Slower, cheaper variant of the hopper  |Slower, cheaper variant of the hopper|  更慢，但更便宜的漏斗替代品|运输缓慢但造价低廉的漏斗代用品|
|     Fancy Item Frame   |    Fancy Item Frame     |   平滑展示框   |   精美物品展示框|
|  ${material} Item Frame|   ${material} Item Frame|  ${material}展示框| ${material}物品展示框|
| Explodes! Safe for mining<sup>1</sup> |       Explodes! Safe for mining     |  爆破！一种安全的挖矿方法。| 爆破，一种安全的挖矿方式！    |
|              Slimesling               |              Slimesling             | 史莱姆弹弓| 黏液弹弓|
|             Slime Boots               |               Slime Boots           | 史莱姆靴子| 黏液靴子|
|Makes you bounce when landing<sup>1</sup>|   Makes you bounce when landing   |提供摔落保护以及粘液的弹跳效果|让你落地时再次弹跳起来|
|Piggybackpack|Piggybackpack|肩托式背包|肩驮架|
|Right-click mobs to pick them up|Right-click mobs to pick them up|右击怪物把它们背起来|右击生物可以驮起它们|
|           Crafting Station            |      Crafting Station                        |  合成站  |    工作站  |
|          Part Builder                 |             Part Builder                     | 部件加工台|  部件制造桌|
|   Shovel      |    Shovel    | 铲  | 锹   |
|   Kama        |    Kama      | 园艺镰 | 短刃镰  |
| Broadsword    | Broadsword  | 大剑   | 阔剑  |
| ${type} Slimy Dirt | ${type} Slimy Dirt | ${type}史莱姆泥土| ${type}黏性泥土 |
| ${type} Slimy Grass | ${type} Slimy Grass | ${type}史莱姆草方块| ${type}黏性草方块 |
|So soft! Lets you jump higher|So soft! Lets you jump higher|很柔软，可以让你跳的更高| 好软啊！能让你跳得更高|

Note 1: these are tooltips.

File: src/main/resources/assets/tconstruct/lang/zh_cn.json (hunk 2 above)
Comment (KnightMiner) [reply src/main/resources/assets/tconstruct/lang/zh_cn.json:95]: The block shows from two cases: throwing a glowball, and from a modifier on tools. The modifier has nothing to do with glowballs. There is a reason we call the block simply "glow" in english and not "glowball"

File: src/main/resources/assets/tconstruct/lang/zh_cn.json (hunk 1 above)
Comment (ETStareak) [reply src/main/resources/assets/tconstruct/lang/zh_cn.json:907]: I have explained it in #4173 ,we translate Slime/Slimy life-form to “史莱姆”, and Slime/Slimy metrial to “黏液”, to make it consistent with vanilla Minecraft.

#4173 中提到了，我们将生物性质的Slime/Slimy翻译成“史莱姆xx”，将物品性质的Slime/Slimy翻译成“黏液xx”，从而与原版保持一致。

File: src/main/resources/assets/tconstruct/lang/zh_cn.json (hunk 2 above)
Comment (ETStareak) [reply src/main/resources/assets/tconstruct/lang/zh_cn.json:95]: No matter “荧光” or “辉光” is not a noun in Chinese, they shouldn't be used as name of a block. I need a better expression I think.

Comment (KnightMiner) [reply src/main/resources/assets/tconstruct/lang/zh_cn.json:95]: The definition of glow in english which fits here is "a steady radiance of light." If you can find a noun with a similar meaning that is not used for torches/glowstone that could work.

**Comment by eav-eav-eav** [comment]
Any progress recently? Or I'm willing to help with the [guide books](https://github.com/SlimeKnights/TinkersConstruct/tree/1.16/src/main/resources/assets/tconstruct/book).

**Comment by KnightMiner** [comment]
As a heads up, Tinkers' Construct is going to be done with 1.16 very shortly. I get that you guys have strong standards for translations, but I would rather have a lower quality translation in the mod than one that is not finished.

You are free to retarget finish this before the final 1.16 build or retarget 1.18, but if I don't see any significant progress on this then do not be offended if I accept a different Simplified Chinese translation.

**Comment by hec9527** [comment]
Will the next release merge this PR

**Comment by hec9527** [comment]
Will this PR be merged in the next release?

**Comment by KnightMiner** [comment]
Until the author marks it ready for I will not be merging it. Its possible they will just have to retarget 1.18 as main development is moving there.

**Comment by HaloSense** [comment]
Hi there ETStareak. I am currently working on further Chinese translation of TiC for version 1.16, along with two of my friends. I see your contribution to the repository and would like to ask whether you are still working on it and whether we could merge part of our work with yours. Or we could simply continue together?

ETStareak 你好，我现在在和我的两个朋友一起汉化1.16版本的匠魂。我看到了你的这个pr并且想来问问你是否仍然在做翻译，我们是否可以合并一部分翻译结果，或者一起继续完成后续汉化？

**Comment by KnightMiner** [comment]
Alright, 1.16 got its last major feature release and this is still marked as a draft. Going to say unless this translation will be done by the end of the month, you should target 1.18. I noticed one bugfix I missed so I may do one last release on 1.16 with that fix, any translations need to be ready by then or they will not make it

Also, does #4232 have merit as a translation at all?

**Pull request closed without merging**